jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaLayerConfigurations.java

```diff
@@ -170,7 +185,35 @@ private JavaLayerConfigurations(
     return getLayerEntries(LayerType.EXPLODED_WAR);
   }
 
+  public String getDependencyDefaultExtractionPath() {
```

Comment:
Are these getting defaults or the actual extraction path?

Reply:
I'll remove `Default` from the name.